packages/SketchMorph2-Core.package/MorphicMonet.class/class/initialize.st

```diff
@@ -0,0 +1,3 @@
+class initialization
+initialize
+	TheWorldMenu registerOpenCommand: {'Morphic Monet'. {self. #open}}
```

Comment:
Is it really necessary to have an own method for this single command?

Reply:
I think it's alright to have an extra method for this, but I would call it something like `addToWorldMenu` rather than `initialize`.